integration/js/app/video_test/video_test.html

```diff
@@ -62,6 +62,76 @@ <h2>Join a meeting</h2>
           Unbind Video Element
         </button>
       </form>
+
+       
+      <form id="add-video-tile-form">
+        <button id="add-video-tile-btn" type="submit">
+          addVideoTile
+        </button>
+        <label for="add-video-tile-label">Tile Id: </label>
+        <span id="video-tile-id"></span>
+      </form>
+
+      <form id="has-started-local-video-tile-form">
+        <button id="has-started-local-video-tile-btn" type="submit">
+          hasStartedLocalVideoTile
+        </button>
+        <label for="has-started-local-video-tile-label">Value: </label>
+        <span id="has-started-local-video-tile-boolean"></span>
+      </form>
+
+      <form id="get-local-video-tile-form">
+        <button id="get-local-video-tile-btn" type="submit">
+          getLocalVideoTile
+        </button>
+        <label for="local-video-tile-label">Tile Id: </label>
```

Comment:
Does this only return the Tile ID?

Reply:
It returns VideoTile but I only use id to verify the response.